src/com/yetanalytics/lrs_admin_ui/subs.cljs

```diff
@@ -160,3 +161,133 @@
  :<- [:oidc/show-local-login?]
  (fn [show-local-login? _]
    show-local-login?))
+
+;; Status Dashboard
+(reg-sub
+ :status/enabled?
+ (fn [db _]
+   (::db/enable-admin-status db false)))
+
+(reg-sub
+ :db/status
+ (fn [db _]
+   (::db/status db)))
+
+(reg-sub
+ :status/params
+ :<- [:db/status]
+ (fn [status _]
+   (:params status)))
+
+(reg-sub
+ :status.params/timeline-unit
+ :<- [:status/params]
+ (fn [params _]
+   (:timeline-unit params "day")))
+
+(reg-sub
+ :status.params/timeline-since
+ :<- [:status/params]
+ (fn [params _]
+   (:timeline-since params (-> js/Date
+                               .now
+                               ;; two weeks ago
+                               (- 12096e5)
+                               (js/Date.)
+                               .toISOString))))
+
+(reg-sub
+ :status.params/timeline-since-local
+ :<- [:status.params/timeline-since]
+ (fn [since _]
+   (u/utc->local-datetime since)))
+
+(reg-sub
+ :status.params/timeline-until
+ :<- [:status/params]
+ (fn [params _]
+   (:timeline-until params (.toISOString (js/Date.)))))
+
+(reg-sub
+ :status.params/timeline-until-local
+ :<- [:status.params/timeline-until]
+ (fn [until _]
+   (u/utc->local-datetime until)))
+
+(reg-sub
+ :status/data
+ :<- [:db/status]
+ (fn [status _]
+   (:data status)))
+
+(reg-sub
+ :status.data/statement-count
+ :<- [:status/data]
+ (fn [data _]
+   (:statement-count data)))
+
+(reg-sub
+ :status.data/actor-count
+ :<- [:status/data]
+ (fn [data _]
+   (:actor-count data)))
+
+(reg-sub
+ :status.data/last-statement-stored
+ :<- [:status/data]
+ (fn [data _]
+   (:last-statement-stored data)))
+
+(reg-sub
+ :status.data/last-statement-stored-locale
+ :<- [:status.data/last-statement-stored]
+ (fn [last-stored _]
+   (.toLocaleString (js/Date. last-stored))))
```

Comment:
There should probably be a special case when `last-stored` is `nil`. When you load up a fresh UI right now, you get "Last Statement Stored: 12/31/1969, 7:00:00 PM," which seems kind of dumb.

Reply:
https://github.com/yetanalytics/lrs-admin-ui/pull/39/commits/7362b7cb600408df2e664077b3021c3ec3be3abf